src/false_positives.h

```diff
@@ -0,0 +1,118 @@
+/**********************************************************************
+ * Copyright (c) 2020 Pieter Wuille, Greg Maxwell, Gleb Naumenko      *
+ * Distributed under the MIT software license, see the accompanying   *
+ * file LICENSE or http://www.opensource.org/licenses/mit-license.php.*
+ **********************************************************************/
+
+#ifndef _MINISKETCH_FALSE_POSITIVES_H_
```

Comment:
Does this file need to be added to `sources.mk`?

Reply:
cc @naumenkogs

Reply:
I've been troubleshooting latest [travis failures](https://travis-ci.org/github/bitcoin/bitcoin/builds/680361425?utm_source=github_status&utm_medium=notification) and i'm wondering whether this is the reason...